guardian/amiable
# chore: Bump @guardian/cdk 49.0.0 -> 49.4.0

## What does this change?

Update GuCDK to https://github.com/guardian/cdk/releases/tag/v49.4.0. This has been tested in cdk-playground as part of the PR review process, but updating AMIable to exercise this change on a real repository to get some immediate feddback before others start using it.

## How to test

Deploy in CODE, ensure deployment is successful and that the UI is reachable. Review EC2 instance configuration in the AWS console to ensure it matches expectations (tags, SGs etc).

## How can we measure success?

Testing is successful, this change is deployed to PROD.

## Have we considered potential risks?

Deployment fails for some reason associated with the GuCDK release, rollback and remediate in https://github.com/guardian/cdk repostory.
